packages/donation-form/src/donation-form-controller.ts

```diff
@@ -434,14 +434,20 @@ export class DonationFormController extends LitElement {
 
         .donation-form-controller-container .braintree-row {
           display: flex;
+          margin-top: -1px;
```

Comment:
gotta ask why this Dom forced your hand like this?

Reply:
The way we lay out the form fields, we want a 1px line between them, but since each field has its own borders, it causes a double border. This `-1px` margin causes the fields to overlap each other by 1px so the double border looks like one.

<img width="373" alt="Screen Shot 2020-10-27 at 12 07 01 PM" src="https://user-images.githubusercontent.com/51138/97349967-5e629700-184d-11eb-903f-cb5bf8c85c48.png">

It does cause some weirdness when we do error states so during the error state, I use an inner shadow to add an extra pixel of border, otherwise the red border disappears.

<img width="317" alt="Screen Shot 2020-10-27 at 12 11 00 PM" src="https://user-images.githubusercontent.com/51138/97350068-84883700-184d-11eb-8351-3344d6167e15.png">